### PR TITLE
9005 changes for search callback

### PR DESCRIPTION
**Pull request by jordiwes**
*Issue #:* /bcgov/entity#9005
/bcgov/entity#10462

*Description of changes:*

- add call for large search
- remove letter validation from aircraft search

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

**Review by doug-lovett — APPROVED** [review]
looks good, I need to validate my life too.

File: ppr-ui/tests/unit/SearchValidation.spec.ts [hunk 1]
```diff
@@ -522,23 +522,8 @@ describe('Aircraft validation', () => {
...
-    // letters only
-    wrapper.vm.$data.searchValue = 'abcd1'
-    await Vue.nextTick()
-    expect(wrapper.vm.$data.validations?.searchValue?.message).toBeDefined()
-    await Vue.nextTick()
-    const messages2 = wrapper.findAll('.v-messages__message')
-    expect(messages2.length).toBe(1)
-    expect(messages2.at(0).text()).toContain('Must contain letters only')
-    // special chars
-    wrapper.vm.$data.searchValue = '10@'
```
Comment (kialj876) [inline ppr-ui/tests/unit/SearchValidation.spec.ts:535]: Hey steph, I think it still needs to verify against special characters

File: ppr-ui/src/utils/validation-helper.ts [hunk 2]
```diff
@@ -173,10 +173,7 @@ export function validateSearchRealTime (searchState: partialSearchState): Search
     case UISearchTypes.AIRCRAFT:
       searchValue = searchValue?.replace(dash, '')
       if (searchValue?.length > 25) {
-        validation.searchValue.message = 'Maximum 25 letters'
-      }
-      if (!lettersOnly.test(searchValue)) {
```
Comment (kialj876) [inline ppr-ui/src/utils/validation-helper.ts:178]: Just need to add the special char check here instead of letters only

**Review by kialj876 — COMMENTED** [review]
Hey Steph, see below. They won't want special chars allowed for aircraft search

File: ppr-ui/src/utils/validation-helper.ts (hunk 2 above)
Comment (jordiwes) [reply ppr-ui/src/utils/validation-helper.ts:178]: Patrick verified special chars allowed

**Review by kialj876 — APPROVED** [review]

**Pull request merged**